### PR TITLE
Update configtx dependency and integration test

### PR DESCRIPTION
**Pull request by wlahti**
#### Type of change

- Bug fix
- Test update

#### Description

- Bump fabric-config/configtx dependency to v0.0.4.
- Update configtx integration test to use nwo.Broadcast for submitting envelopes.
- Add tests to update orderer and other application configuration.

#### Related issues

[FAB-17956](https://jira.hyperledger.org/browse/FAB-17956)

### CODE REVIEW

File: integration/configtx/configtx_test.go
Comment (sykesm) [inline integration/configtx/configtx_test.go:117]: This seems to be associated with the previous commit, right? If we bump the dependency without this change, will the code build? If not, this should be pulled into the previous commit.

Our CI (for better or worse) does not build and test each commit independently so it's easy to have a broken commit that could bite us during bisects or other postmortem activities.

Comment (wlahti) [reply integration/configtx/configtx_test.go:117]: Good to know. Squashed the two commits together.

File: integration/configtx/configtx_test.go
Comment (sykesm) [inline integration/configtx/configtx_test.go:159]: Did you really want to use `port + 1` or get another reserved port from the network?

**Review by sykesm — APPROVED** [review]
LGTM; I'll let you decide if adding a port to a possibly active server is okay and rely on @caod123 to merge as appropriate.

File: integration/configtx/configtx_test.go
Comment (caod123) [reply integration/configtx/configtx_test.go:159]: I believe the idea was that this port wasn't actually going to be used for anything hence no need to get another reserved port. I think this is fine as is.

**Pull request merged**
